save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ExecutionController.kt

```diff
@@ -112,7 +114,9 @@ class ExecutionController(private val executionService: ExecutionService,
         }
         .flatMap { organization ->
             projectService.findWithPermissionByNameAndOrganization(authentication, name, organization.name, Permission.READ).map {
-                executionService.getExecutionDtoByNameAndOrganization(name, organization).reversed()
+                executionService.getExecutionByNameAndOrganization(name, organization).map {
+                    it.toDto().copy(contestName = lnkContestExecutionService.findContestByExecution(it)?.name)
```

Comment:
Maybe we can check `testingType` of execution here and perform query only if `testingType == CONTEST_MODE`?

Reply:
Done